bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/proposal/ProposalView.scala

```diff
@@ -114,54 +129,55 @@ class ProposalView(advisor:ShellAdvisor) extends BorderPanel with BoundView[Prop
     add(remove, BorderPanel.Position.East)
 
     // Panel for the buttons
-    object selectPanel extends BorderPanel {panel =>
+    lazy val selectPanel = new BorderPanel {panel =>
```

Comment:
were there initialization issues or something?

Reply:
Yes, they are called on the setup of the parent ui control, when they become a `val` (as we have more than one `selectPanel`) the initialization fails unless we declare it `lazy`